var/spack/repos/builtin/packages/py-fastai/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
```

Comment:
```suggestion
# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
```

Reply:
Don't do this, it will cause the tests to fail.

Reply:
Apologies.  I didn't realize we were checking that.